### PR TITLE
Fix build on FreeBSD

### PR DESCRIPTION
**Pull request by MikaelUrankar**
Fix:
```
error[E0425]: cannot find value `get_shell_ffi` in this scope                                                                                                              
  --> src/utils.rs:71:48                                                                                                                                                   
   |                                                                                                                                                                       
71 |     let shell = env::var("SHELL").ok().or_else(get_shell_ffi);                                                                                                        
   |                                                ^^^^^^^^^^^^^ not found in this scope
```

### CODE REVIEW

**Review by robinvd — APPROVED** [review]
i cant test on freebsd but it looks fine!

**Comment by xanderio** [comment]
I can confirm that this change works on FreeBSD

**Comment by robinvd** [comment]
Thx @xanderio always helpful to have a second opinion! now we just need one more maintainer to approve, i have requested a review

**Review by JojiiOfficial — APPROVED** [review]

**Pull request merged**
